src/content/app/species-selector/views/species-manager/SpeciesManager.tsx

```diff
@@ -33,7 +34,9 @@ const SpeciesManager = () => {
   return (
     <ModalView onClose={onClose}>
       <div className={styles.container}>
-        <div style={{ height: '60px' }}>Here will be the dropdown</div>
+        <div style={{ height: '60px' }}>
```

Comment:
The inline style is a remnant of the first commit with placeholders. At this point, we probably want to replace the inline style with a CSS class.